### DISCUSSION TITLE
organisation are flagged, but no response why!!

### DISCUSSION THREAD

**Discussion by o3-shop-user** [community/community#50050] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi,

our organisation was flagged. We got only this automatic text: "Because of that, your organization is hidden from the public. If you believe this is a mistake, [contact support](https://github.com/contact) to have your organization’s status reviewed."

We have no idea why. I opened a ticket at the link "contact support". But after serveral days, no one answered me.
What kind of support is this when no one responds to such a serious matter in a timely manner?

**Comment by Anmol-Baranwal** [comment, marked as answer]
I'm sorry to hear that you are facing difficulties with Github organization status. I can suggest some steps that you can take to get help with this issue:

1. Follow up on your support ticket: If you haven't received a response to your support ticket, you can try following up on it to check the status of your request.

2. Contact Github Support via other channels: Github provides several channels for support, including email, live chat, and social media. You can try reaching out to them through one of these channels to get a quicker response.

3. Check for possible violations: Github may have flagged your organization for violating its terms of service. You can review the guidelines to ensure that your organization is not in violation of any of them.

4. Review your organization settings: Review your organization settings to ensure that they are correctly configured and aligned with Github's policies.

I hope these steps help you get the assistance you need to resolve the issue with your Github organization status.

**Reply by Anmol-Baranwal** [reply]
@o3-shop-user 
I just wanted to check in and see if you had any further questions or concerns about my answer on Github. If you have no further doubts or if your doubts have been resolved, would you mind marking my answer as accepted?

It would be greatly appreciated and it will also help other users who might be looking for a solution to a similar issue.

**Reply by o3-shop-user** [reply]
We still have no idea why our account got banned. Since we don't use a pay plan, there doesn't seem to be any other contact option for us than a ticket. And Github doesn't respond to that.

**Reply by o3-shop-user** [reply]
Still, I closed the ticket as there seems to be no other option but to wait. This makes it clear to us that Github has no interest in clarifying such cases. Sure, you get what you pay for.

**Reply by Anmol-Baranwal** [reply]
It's painful to hear, I still hope your issue gets resolved quickly.

**Reply by laurensius** [reply]
i faced the same thing. how to solve this problem?

**Reply by sliyarli** [reply]
> i faced the same thing. how to solve this problem?

did you find something?

**Reply by THE-AUDIO** [reply]
mee to i have the same probleme

**Comment by khemraj-h** [comment]
im having this same issue, also they removed all the images I had and our readme for public AND PRIVATE for the organization, would really appreciate some clarity on this

**Reply by Paronify** [reply]
I am having the same issue, the organization was flagged and my collaborators can't work.

**Comment by tiagox** [comment]
My organization it's being flagged since May 27, 2024. **More than 2 months**.

We have not received any email notification with a reason for the flag.

We have no idea what might had happened.

It's disgusting that GitHub turns their backs on their users.

We're actually migrating to GitLab.

I wonder if anyone could resolved this kind of block. I know this is the channel to send requests... but it's literally impossible to contact support and get at least an ACK from them.

**Reply by gsw85** [reply]
found out gitlab is actually not bad

**Reply by tiagox** [reply]
GitLab is great, it's been working like a charm. I'd like to recover my organization back... still no response from "The Support Team"... if it's like even a real thing.

**Comment by TahmidFarzan** [comment]
I am facing same problem. it say 'Your account has been flagged.'
I have rise a ticket 'https://support.github.com/ticket/personal/0/2916054' but no answer is coming form them.
I am facing problem. some one tell me what should i do ?
If you @github-staff see this, Please help.

**Comment by sindicatoesp** [comment]
We are experiencing a similar issue with the UNISDR GitHub organization (United Nations Office for Disaster Risk Reduction).

Our organization was recently flagged and is now hidden from public visibility. We have already submitted a support request and are awaiting review.

The organization hosts public open-source projects, including the [DELTA resilience](https://github.com/unisdr/delta).

Organization: UNISDR
Repository example: DELTA
Project website: DELTA Resilience

At the moment, the restriction is affecting collaboration and visibility of our open-source work.

We understand that GitHub may need time to investigate potential policy or security concerns and we are fully willing to provide any verification required regarding organizational ownership and legitimacy.

We are posting here primarily to:
- determine whether others have encountered similar situations,
- understand if there are any additional steps we should take,
- and increase visibility of our pending review request.

If a GitHub staff member or moderator is able to advise on appropriate next steps, we would greatly appreciate the guidance.

Thank you.
